packages/block-library/src/cover/edit.native.js

```diff
@@ -106,18 +107,23 @@ const Cover = ( {
 	);
 
 	useEffect( () => {
+		let isCurrent = true;
```

Comment:
I was wondering if we could rename these `isCurrent` to `isMounted` it took me a bit to understand what it was doing 😅 what do you think?

Reply:
Agreed, the concept of "mounted" is the most common case for _why_ one would need to cancel an effect, but it [doesn't cover all scenarios](https://twitter.com/ryanflorence/status/1129566998864715776) (e.g. a changed Hook dependency could invalidate the callback). We could name this particular case `isMounted`, because it definitely applies here, but I wonder if a consistent (albeit more generic) name is beneficial. That is why I opted for `isCurrent`. WDYT?

Reply:
Loved that thread with naming suggestions 😆 `isCurrent` sounds good to me now, thanks for explaining!